3_javascript/javascript3/week2/scripts.js

```diff
@@ -0,0 +1,85 @@
+let fromCurrency = 'EUR';
+let toCurrency = 'DKK';
+let amount = 0;
+
+document.addEventListener('DOMContentLoaded', () => {
+  const inputAmount = document.getElementById('amount');
+  const fromCurrencySelect = document.getElementById('fromCurrency');
+  const toCurrencySelect = document.getElementById('toCurrency');
+  const clickConvert = document.getElementById('convert');
+
+  function populateCurrencyDropdowns() {
+    const currencyPairs = [
```

Comment:
The currencyParis array should have been populated from the API instead of being hardcoded.